test/nuts/plugin-install.nut.ts

```diff
@@ -67,6 +73,12 @@ describe('plugins:install commands', () => {
 describe('plugins:install commands', () => {
   before(async () => {
     session = await TestSession.create();
+    await fs.writeJson(
+      path.join(session.homeDir, '.sfdx'),
```

Comment:
@mshanemc I thought you said this file needed to be named `.sfdx/acknowledgedUsageCollection.json`?

Reply:
yeah, oops